### PR TITLE
customWait Util

### PR DESCRIPTION
**Pull request by qingyuanre1**
customeWait Util done.

### CODE REVIEW

**Comment by mmmnation** [comment]
good job

**Review by mmmnation — APPROVED** [review]
good job

**Review by yadikaer — APPROVED** [review]
good use of code

**Review by huidong198706 — APPROVED** [review]
good good study day day up.

File: src/test/java/selenium/customWait.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package selenium;
+
+public class customWait {
+
+	static void customeWait(int second) {
```
Comment (Chenli2022) [inline src/test/java/selenium/customWait.java:5]: Good job!

**Review by Chenli2022 — COMMENTED** [review]
well done

**Pull request merged**